src/ansys/mapdl/core/mapdl.py

```diff
@@ -648,6 +648,17 @@ def force_output(self):
         """
         return self._force_output(self)
 
+    @property
+    def save_selection(self):
+        """Save selection
+
+        Save the current selection (nodes, elements, keypoints, lines, areas,
+        volumes and components) before entering in the context manager, and
+        when exit returns to that selection.
+
+        """
+        return self._save_selection(self)
```

Comment:
I didn't know you have to use self to access nested classes

Reply:
Mmmhhh... well.. self is Mapdl, which is passed later to the `__init__` .. so I guess you don't need it?